src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -277,12 +277,14 @@ const DataAccessRequestApplication = (props) => {
     }
 
     formData.researcher = isNil(researcher) ? '' : researcher.displayName;
+    formData.piName = props.existingDarsReadOnlyMode ? formData.piName : formData.researcher;
+    formData.piEmail = props.existingDarsReadOnlyMode ? formData.piEmail : (isNil(researcher) ? '' : researcher.email);
```

Comment:
Can we pull this out of props directly instead? It makes it easier to track use inside components.